pkg/model/postgresql_aws_periodic_backup.go

```diff
@@ -0,0 +1,125 @@
+package model
+
+import (
+	"github.com/keycloak/keycloak-operator/pkg/apis/keycloak/v1alpha1"
+	v13 "k8s.io/api/batch/v1"
+	"k8s.io/api/batch/v1beta1"
+	v1 "k8s.io/api/core/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func PostgresqlAWSPeriodicBackup(cr *v1alpha1.KeycloakBackup) *v1beta1.CronJob {
+	return &v1beta1.CronJob{
+		ObjectMeta: v12.ObjectMeta{
+			Name:      cr.Name,
+			Namespace: cr.Namespace,
+			Labels: map[string]string{
+				"app":       ApplicationName,
+				"component": PostgresqlBackupComponent,
+			},
+		},
+		Spec: v1beta1.CronJobSpec{
+			Schedule: cr.Spec.AWS.Schedule,
```

Comment:
Has it been tested if this is in an incorrect format?

I would assume the CronJob resource creation would hopefully fail which we can pick up on in if the resources are ready

Reply:
In this case, the reconciliation will fail and in the status field, you'll get this:

```
Status:
  Message:  CronJob.batch "example-keycloakbackup" is invalid: spec.schedule: Invalid value: "*/2 whooops, someone ate all the stars!": Expected exactly 5 fields, found 7: */2 whooops, someone ate all the stars!
  Phase:    failing
  Ready:    false
```

Reply:
excellent, thank you